script2.js

```diff
@@ -0,0 +1,123 @@
+//Домашнее задание 2
+/*Задание 1
+var a = 1, b = 1, c, d;
+c = ++a; alert(c);           // 2
+d = b++; alert(d);           // 1
+c = (2+ ++a); alert(c);      // 5
+d = (2+ b++); alert(d);      // 4
+alert(a);                    // 3
+alert(b);                    // 3
+//Ответ:  На ответ влияет положение оператора (значение переменной одинаково, но в операции b++ участвует значение до инкремента). 
+*/
+
+/*Задание 2
+var a = 2;
+var x = 1 + (a *= 2);
+//Ответ: х равен 5
+*/
+
+//Задание 3
+var a = +prompt('Введите первое число');
+var b = +prompt('Введите второе число');
+
+if (a >= 0 && b >= 0) {
+    var diff  = a - b;
+    alert('Разность положительных чисел: ' + diff);
+} else if (a < 0 && b < 0) {
+    var op = a * b;
+    alert('Произведение отрицательных чисел: ' + op);
+} else if (a >= 0 && b < 0 || a < 0 && b >=0){
+    var sum = a + b;
+    alert('Сумма положительного и отрицательного числа: ' + sum);
+}
+
+//Задание 4
+//Если необходим вывод от а до 15, значит убираем break;
+a = Math.round(Math.random() * 15);
+alert('Загадано число ' + a);
+switch (a){
+    case 0:
+    alert('0');
+    case 1:
+    alert('1');
+    case 2:
+    alert('2');
+    case 3:
+    alert('3');
+    case 4:
+    alert('4');
+    case 5:
+    alert('5');
+    case 6:
+    alert('6');
+    case 7:
+    alert('7');
+    case 8:
+    alert('8');
+    case 9:
+    alert('9');
+    case 10:
+    alert('10');
+    case 11:
+    alert('11');
+    case 12:
+    alert('12');
+    case 13:
+    alert('13');
+    case 14:
+    alert('14');
+    case 15:
+    alert('15');
+    break;
+}
+
+//Задание 5
+function plus(x,y){
+    return x + y;
+}
+
+function minus (x1,y1){
```

Comment:
Между операторами и операндами нужен пробел
```suggestion
function minus (x1, y1){
```